pkg/storage/unified/search/bleve_mappings.go

```diff
@@ -24,11 +24,12 @@ func getBleveDocMappings(_ resource.SearchableDocumentFields) *mapping.DocumentM
 	}
 	mapper.AddFieldMappingsAt(resource.SEARCH_FIELD_NAME, nameMapping)
 
-	// for sorting by title
-	titleSortMapping := bleve.NewKeywordFieldMapping()
-	mapper.AddFieldMappingsAt(resource.SEARCH_FIELD_TITLE_SORT, titleSortMapping)
+	// for filtering/sorting by title full phrase
+	titlePhraseMapping := bleve.NewKeywordFieldMapping()
```

Comment:
So this mapping indexes the literal string, and the title mapping applies whatever the default analyzer is to the text first?

Reply:
we are indexing 2 "title" fields now ( title is maybe not necessary ).  but i'm not sure yet if there is a scenario where we need "title" that uses the default analyzer.

Reply:
sounds good 👍